content/en/_index.md

```diff
@@ -26,25 +26,19 @@ IAM, Kubernetes, Elastic Load Balancing etc -- that's what Opta does!
   </iframe>
 </p>
 
-## Who is it for?
+## Why use Opta
+Infrastructure as code (IaC) has rapidly become the standard for provisioning and managing Infrastructure and 
+for the right reasons! But the leading IaC tools are still complicated to use and require deep Cloud/Infrastructure
+expertise. Opta was conceptualized to help address this complexity. Opta is a simpler IaC framework with best practices
+built-in. It enables users to set up automated, scalable and secure infrastructure without being a cloud expert or 
+spending days figuring out cloud minutiae!
 
-Opta is designed for folks who are not Infrastructure or Devops experts but still want to build amazing,
-scalable, secure Infra in the cloud. Majority of Opta's users are early stage startups who use it for their
-dev/staging/production environments.
+We are confident it can drastically reduce the cloud complexity and devops headaches of most fast moving 
+organizations. It is already being used by companies - big and small :)
 
-If you'd like to try it out or have any questions - feel free to join our [Slack](https://slack.opta.dev/)!
+To read more about the vision behind Opta, check out this [blog post](https://blog.runx.dev/infrastructure-as-code-for-everyone-7dad6b813cbc).
 
-## What you get with Opta
```

Comment:
you sure you wanna torch this section?

Reply:
Yeah - this information is already covered in the README.